front-end/package.json

```diff
@@ -18,52 +18,100 @@
     "Thibaut Sardan"
   ],
   "dependencies": {
+    "@apollo/client": "^3.7.0",
     "@ardatan/graphql-tools": "^4.1.0",
-    "@polkadot/api": "^6.3.2-7",
-    "@polkadot/extension-dapp": "^0.40.3",
-    "@polkadot/extension-inject": "^0.40.3",
-    "@polkadot/react-identicon": "^0.85.5",
-    "@polkadot/ui-settings": "^0.85.5",
-    "@polkadot/util": "^7.4.1",
-    "@polkadot/util-crypto": "^7.4.1",
+    "@artsy/fresnel": "^6.1.0",
+    "@graphql-codegen/cli": "^2.13.3",
+    "@graphql-codegen/typescript": "^2.7.3",
+    "@graphql-codegen/typescript-operations": "^2.5.3",
+    "@graphql-codegen/typescript-react-apollo": "^3.3.3",
+    "@polkadot/api": "^9.6.1",
+    "@polkadot/api-augment": "^9.6.1",
+    "@polkadot/extension-dapp": "^0.44.6",
+    "@polkadot/extension-inject": "^0.44.6",
+    "@polkadot/react-identicon": "^2.9.12",
+    "@polkadot/ui-settings": "^2.9.12",
+    "@polkadot/util": "^10.1.11",
+    "@polkadot/util-crypto": "^10.1.11",
     "@polkassembly/util": "^0.2.0",
-    "@xstyled/styled-components": "^2.1.0",
+    "@tinymce/tinymce-react": "^4.2.0",
+    "@types/jest": "^26.0.20",
```

Comment:
Some dependencies are to be shifted to devDeps

Reply:
no lets keep everything in dependencies. Dev is for servers where you dont want everything on the server. We dont have servers we use build